pyriemann/utils/mean.py

```diff
@@ -363,6 +363,66 @@ def mean_alm(covmats, tol=1e-14, maxiter=100,
         return C_iter.mean(axis=0)
 
 
+def mean_power(covmats, p, *, sample_weight=None, zeta=10e-10):
```

Comment:
What is the * here? Shouldn't this be explained in the doc?

Reply:
Parameters after * are keywords only.